package.json

```diff
@@ -62,6 +62,22 @@
       "stylelint",
       "stylelint/jest"
     ],
+    "overrides": [
+      {
+        "files": [
+          "**/*.mjs"
+        ],
+        "plugins": [
+          "eslint-plugin-import"
+        ],
+        "rules": {
+          "import/extensions": [
+            "error",
```

Comment:
It should probably be a warning for now so that it won't add noise to new PRs.

Reply:
I think `error` is correct here, because extension *is* required. Without out code is invalid and won't work.